packages/kpn-style/README.md

```diff
@@ -0,0 +1,15 @@
+# KPN STYLES
```

Comment:
`# KPN Style`

I think this file should be expanded a lot. This README is too limited for use in the next release, as it would be the only documentation bundled with this package. The current architecture of the repo focuses on using the README and CONTRIBUTE md files as a double whammy: it also provides the content for the portal/react app. This new structure necessitates either duplication or a smart trick. Some more thought at least.

Reply:
I think the way we see documentation has to change.

On one hand we have the root `README.md` for the whole project, which can still be used in the `portal/react app`. 
The one inside `kpn-style/README.md` has to be just a small chunck with very little information. Because people usually don't go to the `packages/**/README.md`, they either go to the root `README` or they go to the docs directly. Sometimes people go to the `package/**/README.md` to find specific about that package, (install, docs specifically for that project, but not more).

Some examples:

[create react app readme](https://github.com/facebook/create-react-app) and [create react app **package** readme](https://github.com/facebook/create-react-app/tree/master/packages/create-react-app)

[react readme](https://github.com/facebook/react) and [react **package** readme](https://github.com/facebook/react/tree/master/packages/react).

What do you think about it?